packages/front-end/components/Experiment/Results.tsx

```diff
@@ -1,24 +1,24 @@
+import { ExperimentSnapshotInterface } from "@back-end/types/experiment-snapshot";
 import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
-import React, { FC, useEffect } from "react";
-import dynamic from "next/dynamic";
 import { DifferenceType, StatsEngine } from "back-end/types/stats";
-import { getValidDate, ago } from "shared/dates";
+import dynamic from "next/dynamic";
+import { FC, useEffect } from "react";
 import { DEFAULT_STATS_ENGINE } from "shared/constants";
+import { ago, getValidDate, relativeDate } from "shared/dates";
```

Comment:
I'm not sure if this diff is because of my linter or if it's configured in the repo. Could use some configuration help to make sure I'm not adding unnecessary diffs along with my changes

Reply:
yeah, I believe this will break. Specifically the `@back-end` reference (these aliases only work currently for front-end)

Reply:
actually looks like that may no longer be the case, nevermind :)

Reply:
Not sure if you're using VSCode, but wondering if this is an IDE setting causing the reordering:
https://stackoverflow.com/questions/73302972/how-to-disable-sorting-import-order-by-prettier-in-vscode

Reply:
Yep that was exactly the problem, thanks for grabbing that for me. I guess this is what I get for installing an opinionated extension and not configuring it myself :stuck_out_tongue: